pypeerassets/pautils.py

```diff
@@ -62,7 +62,7 @@ def find_deck_spawns(provider, prod=True):
         else:
             raise NotImplementedError
 
-    if isinstance(provider, Holy) or isinstance(provider, Cryptoid):
+    if isinstance(provider, Cryptoid):
```

Comment:
hrm... maybe i should have updated this to be `if isinstance(provider, Explorer) or isinstance(provider, Cryptoid):` :thinking:

Reply:
mmm... changing it to `if isinstance(provider, Explorer) or isinstance(provider, Cryptoid):` doesn't make much of an impact on the tests at the moment... maybe should just leave it as-is and fix it as needed with the rest of that test suite